src/directives/formly-form.test.js

```diff
@@ -933,12 +933,19 @@ describe('formly-form', () => {
 
       const listener = sinon.spy()
       const expression = sinon.spy()
+      const deepLinkField = getNewField()
+      deepLinkField.key = 'foo.bar'
+      deepLinkField.watcher = {
+        listener: '',
```

Comment:
I don't think that this test is testing what you think it is... I think you're getting code coverage, but the watchExpression here never gets evaluated. Could you add a new test that has a deepLink field like this except for the `expression` uses a spy and asserts that the spy was called properly?